scene/resources/3d/importer_mesh.cpp

```diff
@@ -1098,15 +1102,7 @@ Ref<NavigationMesh> ImporterMesh::create_navigation_mesh() {
 	Ref<NavigationMesh> nm;
 	nm.instantiate();
 	nm->set_vertices(vertices);
-
-	Vector<int> v3;
-	v3.resize(3);
-	for (uint32_t i = 0; i < face_indices.size(); i += 3) {
-		v3.write[0] = face_indices[i + 0];
-		v3.write[1] = face_indices[i + 1];
-		v3.write[2] = face_indices[i + 2];
-		nm->add_polygon(v3);
-	}
+	nm->set_polygons(face_polygons);
```

Comment:
While not focus of this PR this line and the above set_vertices() should be combined and use the newer set_data() to keep both vertices and polygons arrays thread-safe and in sync.